pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1616,6 +1620,47 @@ func configAdapters(ctx *domainContext, config types.DomainConfig) error {
 	return nil
 }
 
+func createMountPointFile(imageHash, mpFileName string, status types.DomainStatus) error {
+	file, err := os.Create(mpFileName)
+	if err != nil {
+		log.Errorf("createMountPointFile: os.Create for %v, failed: %v", mpFileName, err.Error())
+	}
+	defer file.Close()
+
+	appManifest, err := getAppManifest(imageHash)
+	if err != nil {
+		return fmt.Errorf("createMountPointFile: Error while fetching app manfest: %v", err.Error())
+	}
+
+	//Ignoring container image in status.DiskStatusList
+	if (len(status.DiskStatusList) - 1) != len(appManifest.App.MountPoints) {
+		if len(status.DiskStatusList) > len(appManifest.App.MountPoints) {
```

Comment:
Shouldn't this be >=

Reply:
When no. of disks is (strictly) greater than no. of mount-points provided, I added a warning saying we'll be ignoring the extra disks. This warning won't be needed when no. of disk =  no. of mount-points.
I missed to decrement len(status.DiskStatusList) by 1. Corrected that and added comments.